pkgs/os-specific/linux/rtl8852cu/default.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  kernel,
+  bc,
+  nukeReferences,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "rtl8852cu";
+  version = "${kernel.version}-unstable-20240510";
+
+  src = fetchFromGitHub {
+    owner = "morrownr";
+    repo = "rtl8852cu-20240510";
+    rev = "15788c86a7ae14dd74cac7e475b6f4a9953a2c8c";
+    hash = "sha256-nd6SoIG28Y29OXlwofrIqH8UNBVq9/TVsapX+ADuw10=";
+  };
+
+  nativeBuildInputs = [
+    bc
+    nukeReferences
+  ]
+  ++ kernel.moduleBuildDependencies;
+  hardeningDisable = [
+    "pic"
+    "format"
+  ];
+
+  postPatch = ''
+    substituteInPlace ./Makefile \
```

Comment:
Looks like it would be easier to just not use `make install` and do it manually? All we actually want it to do is install a single file, right?